project/src/components/offers-list/offers-list.tsx

```diff
@@ -17,10 +17,7 @@ function OffersList({ offers, className, onOfferCardHover }: OffersListProps) {
 
   return (
     <div className={offersListClassName}>
-      {offers.map((offer) => {
-        const { facilities, host, reviews, ...card } = offer;
-        return <OfferCard key={card.id} info={card} className={className} onOfferCardHover={onOfferCardHover} />;
-      })}
+      {offers.map((offer, idx) => <OfferCard key={idx.toString()} offer={offer} className={className} onOfferCardHover={onOfferCardHover} />)}
```

Comment:
```suggestion
      {offers.map((offer, idx) => <OfferCard key={idx} offer={offer} className={className} onOfferCardHover={onOfferCardHover} />)}
```
не нужно все приводить к строке

Reply:
eslint ругается пишет нельзя использовать idx в качестве key

Reply:
done